enclave/core/malloc.c

```diff
@@ -2,14 +2,11 @@
 // Licensed under the MIT License.
 
 /* The use of dlmalloc/malloc.c below requires stdc names from these headers.
- * Ensure that all corelibc headers are included with std names before
- * including other headers. */
-#define OE_NEED_STDC_NAMES
+ * This define is set by enclave/core/CMakeLists.txt */
```

Comment:
What is "This define" referring to?

Reply:
Oh whoops. I removed the define and now the comment needs context